SMLHelper/Patchers/PingTypePatcher.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Just a personal preference: but get rid of this if you can.
~~using System.Linq;~~